mlflow/models/utils.py

```diff
@@ -541,6 +572,22 @@ def _load_serving_input_example(mlflow_model: Model, path: str):
         return handle.read()
 
 
+def load_serving_example_from_uri(model_uri_or_path: str):
+    """
+    Load serving input example from a model directory or URI.
+
+    Args:
+        model_uri_or_path: Model URI or path to the `model` directory.
+            e.g. models://<model_name>/<model_version> or /path/to/model
+    """
+    serving_input_path = model_uri_or_path.rstrip("/") + "/" + SERVING_INPUT_FILENAME
+    local_serving_input_path = _download_artifact_from_uri(serving_input_path)
+    with open(local_serving_input_path) as handle:
+        result = handle.read()
+    # To avoid including indent in the output
+    return json.dumps(json.loads(result))
```

Comment:
Would `indent` be a problem?

Reply:
Seems so 
<img width="948" alt="image" src="https://github.com/user-attachments/assets/de71ff6c-5622-481e-9a0c-ab2f29c4b6f5">

Reply:
The JSON in the image looks valid.

Reply:
Yes, if you loads it back then the indent is removed

Reply:
Updated!